src/joint.dia.link.js

```diff
@@ -1446,7 +1436,7 @@ joint.dia.LinkView = joint.dia.CellView.extend({
         // if are simulating pointerdown on a link during a magnet click, skip link interactions
         if (evt.target.getAttribute('magnet') != null) return;
 
-        var interactive = _.isFunction(this.options.interactive) ? this.options.interactive(this, 'pointerdown') : this.options.interactive;
+        var interactive = _.isFunction(this.options.interactive) ? this.options.interactive(this) : this.options.interactive;
```

Comment:
I still don't understand what you want me to change here. Can you do merge this PR then do what you want in a new PR?